src/client/pages/Landing/landingPageData.ts

```diff
@@ -25,11 +25,13 @@ type ProductData = {
 
 type ProductsData = Record<Market, ProductData>
 
+const isQuoteCartApiEnabled = process.env.ENABLE_QUOTE_CART_API === 'true'
+
```

Comment:
We know that this is weird, but it's just temporary until we are able to have only the new api!